.travis.yml

```diff
@@ -21,10 +21,6 @@ matrix:
     - env: BUILD=pkglint
       python: 3.8
 
-# Pull all of the history so we can use reno
-git:
-  depth: false
```

Comment:
This option will also be needed for the contributor name filter tests.

Reply:
We should update the comment to explain that, since reno is no longer used.

Reply:
That makes sense. I've restored the git depth configuration and adjusted the comment.